invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/invitations/InvitationActionButtons.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
```

Comment:
Not sure I would keep this file, I think this logic is very tied to the invitations and not sure it can be reused, if not reusable I would move it to `InvitationResultItem.js`. WDYT?

P.S If you want to keep the file please add the header.